### PR TITLE
Adds website field for career opportunities

### LINKED ISSUE #2292: Add button to jobb application as a dedicated field to be a part of "Nøkkelinformasjon"
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I am making a new "Karrieremulighet" and I have to insert an external link to the application on the company's website in the application text.

**Describe the solution you'd like**
I want a dedicated field in the dashboard where I can add a link to an external site, and it is shown a part of "Nøkkelinformasjon". 

**Describe alternatives you've considered**
Anything that makes it simple to add an external link and it is clearly visible to users of the site.

**Additional context**

### PR DESCRIPTION
**Pull request by VigdisIS**
## What kind of a pull request is this?
Fixes #2292 

## Code Checklist

- [x] The code follows dotkom code style 
- [x] The code passes the defined tests
- [ ] I have added tests for the code I added
- [ ] I have provided documentation for the code I added
- [x] The code is ready to be merged

## (Possible) Breaking Changes

- [x] The changes are backwards compatible

- [ ] I have updated the build configuration
- [ ] These changes requires changes to configuration in production 
    - [ ] I have applied the required changes in production
    - [ ] I cannot apply the required changes in production before this is deployed.

## Description of changes
Adds a website field to the model and makes sure it's displayed properly throughout the website.

## Screenshot

Skjermbilde 2019-09-25 kl  20 20 39

### CODE REVIEW

File: assets/careeropportunity/components/InfoOnOpportunity.js [hunk 1]
```diff
@@ -67,6 +67,7 @@ const InfoBox = props => (
 
             <div className="row">
               <div className="col-md-12">
+                <p>Nettsted: <a href={props.website}> {props.website} </a></p>
```
Comment (henrikhorluck) [inline assets/careeropportunity/components/InfoOnOpportunity.js]: Can you add a screenshot of the new design to the PR?

File: apps/authentication/migrations/0042_auto_20190918_1958.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    dependencies = [
+        ('authentication', '0041_auto_20190911_2238'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='onlineuser',
+            name='allergies',
+            field=models.TextField(blank=True, null=True, verbose_name='matallergier/preferanser'),
+        ),
```
Comment (henrikhorluck) [inline apps/authentication/migrations/0042_auto_20190918_1958.py]: Also, I don't think this migration is relevant to the PR, so I would suggest removing it.

File: assets/careeropportunity/components/InfoOnOpportunity.js (hunk 1 above)
Comment (VigdisIS) [reply assets/careeropportunity/components/InfoOnOpportunity.js]: Done.

File: apps/authentication/migrations/0042_auto_20190918_1958.py (hunk 2 above)
Comment (VigdisIS) [reply apps/authentication/migrations/0042_auto_20190918_1958.py]: Done.

**Pull request merged**